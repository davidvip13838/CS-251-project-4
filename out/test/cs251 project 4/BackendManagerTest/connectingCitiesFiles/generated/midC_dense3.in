500
30 48
23 16
2 48
45 50
25 41
47 27
49 12
12 1
25 4
42 39
16 46
14 48
4 28
36 41
24 33
6 1
21 43
3 15
41 29
11 50
10 5
23 3
33 30
6 7
18 7
49 20
10 19
34 48
50 42
47 38
36 0
1 3
29 1
39 11
48 31
11 1
5 45
19 2
17 37
41 14
48 35
47 1
13 36
41 28
27 3
32 20
31 12
30 44
30 16
15 20
47 4
20 29
26 30
12 8
40 11
36 46
42 30
4 49
31 21
7 31
6 0
5 17
38 50
28 32
25 46
16 9
42 33
6 2
0 11
42 5
14 9
6 5
36 49
39 33
34 14
50 17
35 16
25 0
7 29
15 22
43 35
46 35
17 35
11 40
48 41
37 5
15 50
5 26
8 29
25 28
43 18
22 16
15 10
7 7
40 17
5 4
2 38
42 38
16 18
29 13
46 9
5 35
1 35
16 33
47 37
8 22
49 34
0 49
23 0
24 28
39 29
48 29
43 49
0 28
13 8
15 2
20 40
0 10
44 25
36 3
41 1
0 38
44 11
49 39
48 14
27 46
26 1
37 9
42 37
31 4
30 0
40 43
35 14
22 35
6 36
3 50
5 24
12 23
47 43
6 24
29 47
44 18
0 7
23 35
28 38
43 50
43 31
15 0
8 30
7 12
43 34
30 38
29 46
46 30
26 21
32 7
49 48
20 36
5 47
32 47
18 46
49 9
13 40
40 9
29 28
31 41
37 17
15 49
31 5
48 39
28 11
47 42
50 49
0 21
22 38
46 38
49 50
30 42
41 4
23 10
35 32
46 49
35 7
9 19
40 41
12 40
13 39
21 6
40 32
30 27
11 33
8 4
19 45
2 36
20 48
23 9
42 9
26 8
15 45
0 19
21 4
9 5
13 3
17 17
21 49
19 3
50 34
49 44
13 20
37 15
34 50
25 24
12 18
45 33
20 20
3 21
41 26
45 6
27 31
46 5
14 43
19 44
22 37
29 30
20 21
45 21
6 41
40 46
45 19
17 14
1 15
0 47
18 14
48 6
26 17
12 39
50 39
31 22
13 10
38 15
0 23
16 34
44 49
27 37
10 22
18 12
3 39
36 38
24 6
39 28
7 38
25 11
38 3
1 43
19 30
14 23
29 49
9 0
10 25
9 36
48 5
8 45
42 22
34 13
1 31
6 30
9 25
39 19
37 37
29 20
47 34
39 39
17 8
50 3
37 13
41 22
27 9
49 37
33 10
39 4
36 45
32 32
18 40
41 31
20 18
24 12
35 8
46 15
26 11
1 49
49 46
36 11
25 36
31 47
47 23
5 37
27 7
32 39
45 8
5 13
48 28
19 49
15 33
6 28
20 0
20 25
38 49
17 2
28 20
22 42
39 15
24 24
31 24
0 5
24 36
50 26
18 35
46 20
47 5
25 31
42 26
49 33
35 19
22 27
9 13
39 50
50 38
3 24
23 47
32 34
5 32
8 46
23 14
50 48
24 32
13 50
27 49
41 39
45 38
48 4
36 21
32 15
47 48
29 42
48 15
6 14
14 32
26 22
5 44
33 32
14 3
14 7
25 37
14 49
23 23
16 44
3 30
1 42
34 8
6 18
39 9
4 13
14 29
31 9
17 7
32 40
48 3
27 17
40 5
25 20
25 30
41 30
4 39
9 6
23 15
43 10
2 27
3 45
49 0
15 9
38 38
7 3
14 31
31 1
23 49
28 42
45 24
24 34
34 9
22 45
24 31
20 38
4 40
11 7
14 40
30 22
28 33
43 47
21 50
12 30
46 8
5 6
45 45
16 17
20 34
46 41
5 1
12 0
48 23
4 19
21 23
11 41
3 36
11 20
33 19
16 35
4 7
15 47
13 45
32 48
46 25
29 45
41 19
29 16
17 28
43 24
12 33
26 45
33 40
50 28
13 26
6 8
46 10
26 6
39 12
44 48
45 9
4 2
46 32
5 48
26 23
33 17
23 46
18 42
14 13
38 9
27 40
28 21
33 29
49 25
20 6
10 29
2 17
28 44
30 50
12 19
0 1
26 9
46 45
49 7
7 6
18 48
1 16
27 16
39 42
45 1
38 11
11 44
15 34
41 35
12 16
40 7
50 30
44 35
37 33
46 0
27 27
15 23
24 20
39 38
47 18
47 3
0 43
2 19
18 44
33 47
6 40
34 19
2 3
17 0
11 12
26 3
16 49
0 41
14 46
28 10
19 0
8 14
9 31
32 12
