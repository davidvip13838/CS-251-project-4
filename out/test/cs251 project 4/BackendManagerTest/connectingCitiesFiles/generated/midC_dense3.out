797.6851736267935